.travis.yml

```diff
@@ -1,6 +1,20 @@
 language: node_js
 
 node_js:
-    - "8"
+  - '10'
```

Comment:
Does this mean this gulpfile isn't compatible with Node 8?

Reply:
Yes, it does! I always specify and lock the supported versions in the `package.json` (see below). So older versions are not supported, and newer versions are not tested. But 8 is pretty outdated, and I always want to test/support the current LTS, which is 10. Plus, if you install node via let's say `brew install node` you get the latest version, which is 12. So testing against 10 makes more sense 👍 

The `package.json`:

```json
  "engines": {
    "node": ">= 8 <= 12",
    "yarn": ">= 1"
  },
```

Reply:
Then it's time for me to upgrade to Node 10 🤔 But it might be use full to test against multiple Node versions.

Reply:
Sure, makes sense. So how does that work?

Mostly wanted linting and unit testing, since there was none. But a more 'integration'-ish kind of testing is always good. But you would have to install multiple node version, rebuild all modules per version and run all scripts per version, right? Seems complicated to do  just from a config file, but if you've got any pointers 😄

Reply:
And it was added! This PR now triggers 6 builds 🚀 

<img width="1255" alt="Screenshot 2019-09-20 at 14 00 51" src="https://user-images.githubusercontent.com/1607628/65325444-251e1900-dbaf-11e9-87c3-86136e2b40f0.png">